modules/rest-api/src/main/java/org/wso2/carbon/mdm/api/MobileDevice.java

```diff
@@ -127,4 +127,25 @@
 			throw new MDMAPIException(msg, deviceMgtEx);
 		}
 	}
+
+	@GET
+	@Path("count")
+	public int getDeviceCount() throws MDMAPIException {
+
```

Comment:
Fixed in Commit [https://github.com/wso2/product-mdm/commit/50bbb3c45678832c7e0577a9c3e947b4941026b0]